src/sql/sqlops.proposed.d.ts

```diff
@@ -58,6 +58,7 @@ declare module 'sqlops' {
 
 	export class TreeComponentItem extends vscode.TreeItem {
 		checked?: boolean;
+		checkboxDisabled?: boolean;
```

Comment:
In most of the other model view components we use `enabled` as the name for this property so I'd suggest changing things around to use `enabled` with a default of `true` to be consistent with those